feat: Clarify how the Fix issues button works DOCS-700 (#2132)
codacy/docs: docs/repositories/issues.md

!!! info "This section applies to GitHub repositories only"

> If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a modal with a patch that addresses all resolvable issues. Follow the instructions in the modal to apply the patch:

Reviewer: I feel that "Click this button to open a modal" would be the first step of the procedure... 😉 
It might not be so easy to solve, as the button is what indicates that there are available fixes...

Leaving a suggestion below.

```suggestion
If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list.

![Fix issues button](images/issues-fix-issues-button.png)

In this case, Codacy generates a patch that enables you to solve all resolvable issues. To apply this patch to your local branch, do the following:

1.  Click the button **Fix issues** to open a modal with a patch that addresses all resolvable issues.
```

1.  Copy the patch content to your clipboard.
1.  Create a new branch from your default branch.

> 1.  Apply the fixes in your project's root folder, for example with the `pbpaste | patch` command.

Reviewer: What is the "root folder"? I used "local branch" above, as it was the terminology used is the modal.
Please validate all this wording. 🙏

Author: It's the root directory of the repository, so the base directory of the repository (the one printed by [this command](https://stackoverflow.com/questions/957928/is-there-a-way-to-get-the-git-root-directory-in-one-command)).

Developers should be familiar with this (or it's a search away), but I can be more precise:

```suggestion
1.  Apply the fixes from your repository's root directory, e.g., with the `pbpaste | patch` command.
```

> 1.  Review the changes to confirm that they're correct.

Reviewer: Maybe we can reinforce the review here, as the fixes were automatically generated? Feel free to change the wording!

```suggestion
1.  Make sure you review the changes to confirm that they're correct, as they were automatically generated.
```

1.  Push the new branch to your repository.

> ![Fix issues button](images/issues-fix-issues-button.png)

Author: @claudiacarpinteiro I'm on the fence about updating these screenshots. The procedure described in the UI has 3 steps, but while writing the procedure, I realized it should be 5. I'm avoiding a screenshot to prevent confusing users. Thoughts?

Reviewer: I would detach our procedure steps from the steps on the modal. For that, I would suggest that we use only 2 screenshots:

- Current screenshot showing the **Fix issues** button, placed before the procedure steps
- A screenshot of the modal where the user can see the beginning of the patch content and the copy-to-clipboard button, to place right after "Copy the patch content to your clipboard.". Something like this:

<img width="923" alt="image" src="https://github.com/codacy/docs/assets/26439949/d87cbaca-f60a-45ce-933c-e2119781b9a4">

Then we can proceed with our instructions using as many steps as we need (no need to match the modal steps).
